### PR TITLE
fixed snap empty list text #1587445

### PR DESCRIPTION
**Pull request by thibran**
This fixes the bit-size bug: https://bugs.launchpad.net/snappy/+bug/1587445

http://www.zygoon.pl/2016/05/bite-size-bugs-in-snapd.html

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by mvo5** [comment]
add to whitelist

File: cmd/snap/cmd_list.go [hunk 1]
```diff
@@ -59,6 +59,9 @@ func listSnaps(args []string) error {
 	snaps, err := cli.ListSnaps(args)
 	if err != nil {
 		return err
+	} else if len(snaps) == 0 {
+		fmt.Println(i18n.G("No snaps are installed yet. Try 'snap install hello'."))
```
Comment (elopio) [inline cmd/snap/cmd_list.go]: It should be hello-world, instead of hello.

**Comment by elopio** [comment]
Thanks for your contribution, and welcome!

**Comment by thibran** [comment]
Hi all. I signed the CLA and fixed the string, what is left to do to get the patch merged?

**Comment by zyga** [comment]
test this please

**Comment by zyga** [comment]
(this is a comment to our CI system, the author of the patch doesn't have to do anything)

**Comment by zyga** [comment]
Looks good to me, LGTM

File: po/snappy.pot [hunk 2]
```diff
@@ -367,10 +367,12 @@ msgstr  ""
 msgid   "no interfaces found"
 msgstr  ""
 
+msgid   "No snaps are installed yet. Try 'snap install hello'."
```
Comment (zyga) [inline po/snappy.pot]: This should be regenerated with ./update-pot but we can do that easily at release time.

**Comment by mvo5** [comment]
:+1:

**Pull request merged**